client-v2/src/components/FrontsEdit/FrontCollectionsOverview.tsx

```diff
@@ -16,7 +21,12 @@ interface FrontContainerProps {
 
 type FrontCollectionOverviewProps = FrontContainerProps & {
   front: FrontConfig;
+  closeAllCollections: (collections: string[]) => void;
 };
+interface CollectionOverviewCollapseAllButtonProps {
+  front: FrontConfig;
+  closeAllCollections: (collections: string[]) => void;
+}
 
```

Comment:
Not sure what is the best DRY way to write this. 
`type FrontCollectionOverviewProps = FrontContainerProps & CollectionOverviewCollapseAllButtonProps` doesn't feel quite right.

Reply:
This should be fine, I think I'd have defaulted to the other but now you mention it I can see why it would feel odd.